utils/stm.stanza

```diff
@@ -38,6 +38,23 @@ public pcb-struct ocdb/utils/stm/STMPinPropertiesRow :
   generic-props?: True|False
   power-props?: True|False
 
+public defn bank? (r:STMPinPropertiesRow) -> False|Ref :
+  val known-banks = [ "PA", 
+                      "PB", 
+                      "PC", 
+                      "PD" ]
```

Comment:
High pin count STM32 devices will be banks up to `H`.

Reply:
Fixed!